jsettlers.mapcreator/src/jsettlers/mapcreator/control/UndoRedoHandler.java

```diff
@@ -0,0 +1,131 @@
+package jsettlers.mapcreator.control;
+
+import java.util.LinkedList;
+
+import jsettlers.mapcreator.data.MapData;
+import jsettlers.mapcreator.data.MapDataDelta;
+import jsettlers.mapcreator.main.window.EditorFrame;
+
+/**
+ * Handle Undo / Redo actions
+ * 
+ * @author Andreas Butti
+ *
+ */
+public class UndoRedoHandler {
+
+	/**
+	 * Max undo counts
+	 */
+	private static final int MAX_UNDO = 100;
```

Comment:
Usually in my application I have an infinite undo stack.
This code is moved out from Main Window, and not changed. I can remove it.